src/contact_us/index.js

```diff
@@ -1,18 +1,39 @@
 import NavBar from 'main_app/components/NavBar'
 import Footer from 'main_app/components/Footer'
+import { CalendlyEventListener, InlineWidget } from 'react-calendly'
 
-import ContactForm from './components/ContactForm'
 import Article from 'main_app/components/Article'
 
 import classes from './styles.module.scss'
 
 function ContactUs () {
+  function isCalendlyEvent (e) {
+    return e.data.event &&
+    e.data.event.indexOf('calendly') === 0
+  }
+
+  window.addEventListener(
+    'message',
+    function (e) {
+      if (isCalendlyEvent(e)) {
+        if (e.data === 'calendly.event_scheduled') {
+          console.log(e.data)
+        }
+      }
+    }
+  )
```

Comment:
These should not be needed... The library already does this for you with the event `onEventScheduled`. Please remove this code.